.github/workflows/stale.yaml

```diff
@@ -0,0 +1,18 @@
+name: "Maintenance: Close Stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  pull-requests: "write"
+
+jobs:
+  stale:
+    runs-on: "ubuntu-latest"
+    if: github.repository_owner == 'SchemaStore'
+    steps:
+      - uses: "actions/stale@v9"
+        with:
+          close-pr-message: "This PR was closed because it has been stalled for 365 days with no activity. Feel free make a new PR if you wish to continue¶"
+          days-before-pr-stale: 365
+          days-before-pr-close: 0
```

Comment:
I would prefer to use stale+close combination like 350+15 days, so people have time to renew it if needed.